### PR TITLE
Update to 5.8.0.16 and enable access to PipeWire for screen-sharing

### PR DESCRIPTION
**Pull request by patriziobruno**
Screensharing has been proven to work on Fedora Silverblue 34 with X window apps. Example of working apps:
- com.google.Chrome (flathub-beta)
- org.telegram.desktop (flathub)
- com.slack.Slack (flathub)

Wayland applications can't be shared when Zoom runs as flatpak, with or without this change.

### CODE REVIEW

File: us.zoom.Zoom.json [hunk 1]
```diff
@@ -12,10 +12,13 @@
         "--share=ipc",
         "--socket=x11",
         "--socket=pulseaudio",
+        "--socket=session-bus",
         "--share=network",
         "--device=all",
+        "--filesystem=xdg-run/pipewire-0",
         "--filesystem=~/Documents/Zoom:create",
         "--filesystem=~/.zoom:create",
+        "--filesystem=xdg-run/pipewire-0",
```
Comment (nikml) [inline us.zoom.Zoom.json:21]: This line appears to have already been added [above](https://github.com/flathub/us.zoom.Zoom/pull/284/files#diff-29c6733da0fa2e3285ba00b2364abc5f54b65feb2f7209d5dfc4351fcd2d9496R18)

File: us.zoom.Zoom.json [hunk 2]
```diff
@@ -12,8 +12,10 @@
         "--share=ipc",
         "--socket=x11",
         "--socket=pulseaudio",
+        "--socket=session-bus",
```
Comment (muelli) [inline us.zoom.Zoom.json:15]: This gives full access to the user's dbus, no? This is quite a biggie.
I don't feel comfortable giving away access to the full dbus without any justification.
Do you know what it's talking to? Those paths can be given access to.

**Comment by Zeioth** [comment]
Updated to 5.8.0.16, Still getting the 'We only support gnome screen'.

File: us.zoom.Zoom.json (hunk 2 above)
Comment (patriziobruno) [reply us.zoom.Zoom.json]: fair, there's no real need to give it access to the session

**Comment by Iolaum** [comment]
> Updated to 5.8.0.16, Still getting the 'We only support gnome screen'.

As mentioned in [hackernews](https://news.ycombinator.com/item?id=28736088) this is likely because their app is based on a private Gnome d-bus screensharing API.

There is also [this comment](https://news.ycombinator.com/item?id=28740223):

> That's not really a big deal, you could just develop a small shim for that d-bus API if you really needed to. But I agree, it would be better for them to switch to Pipewire.

for whatever it's worth.

**Comment by nedrichards** [comment]
Resolved by https://github.com/flathub/us.zoom.Zoom/commit/fa4a45fd1b83f591c839c07da34378597397d910

**Pull request closed without merging**